### ISSUE TITLE
Why is it called "subtrain"?

### ISSUE THREAD

**Issue by jchoivision** [shshin1210/cv-proj#1]
https://github.com/shshin1210/cv-proj/blob/8a0cac95863eafa41e5414d1354520333a478160/main.py#L70

**Comment by shshin1210** [comment]
I tried to split (full)trainset into sub_train and validation set to use these sets for train and validation before testing test set, but failed using it. So I haven't used codes under `subtrain_do = False \ if subtrain_do:`  (kept the codes just in case)
Therefore, I only trained my full trainset (dataset before splitting into sub_train and val) and did testing on my test set.
But I have successfully used train & val set in PENCIL.py!
